protocol/app/process/market_price_decoder.go

```diff
@@ -0,0 +1,28 @@
+package process
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	pricestypes "github.com/dydxprotocol/v4-chain/protocol/x/prices/types"
+)
+
+// MarketPriceDecoder is an interface for decoding market price transactions, This interface is responsible
```

Comment:
`MarketPriceDecoder` shouldnt this be `UpdateMarketPriceTxDecoder`?

Reply:
That is correct, the name of the type in the docs should be `UpdateMarketPricesDecoder`